### PR TITLE
Add archive ui

### PR DESCRIPTION
**Pull request by jbevan4**
# What
Added the archive UI

# Why
So users can archive a ward
# Screenshots
NA
# Notes
NA

### CODE REVIEW

File: pageTests/admin/archive-a-ward-confirmation.test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { getServerSideProps } from "../../pages/admin/archive-a-ward-confirmation";
+
+// TODO: This needs to be moved once the verifyToken logic is in the container..
```
Comment (StephenThomson) [inline pageTests/admin/archive-a-ward-confirmation.test.js:3]: Is the verifyToken logic now in the container?

File: pageTests/admin/archive-a-ward-success.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { getServerSideProps } from "../../pages/admin/archive-a-ward-success";
+
+jest.mock("../../src/usecases/adminIsAuthenticated", () => () => (token) =>
```
Comment (StephenThomson) [inline pageTests/admin/archive-a-ward-success.test.js:3]: If we aren't moving this now then bring the TODO along too.

File: pages/admin.js [hunk 3]
```diff
@@ -15,26 +15,23 @@ const Admin = ({ wardError, trustError, wards, trust }) => {
...
+    <Layout title={`Ward administration for ${trust.name}`} renderLogout={true}>
       <GridRow>
         <GridColumn width="full">
           <Heading>
             <span className="nhsuk-caption-l">
               {trust.name}
               <span className="nhsuk-u-visually-hidden">-</span>
             </span>
-            Ward admininistration
+            Ward administration
```
Comment (StephenThomson) [inline pages/admin.js:26]: 🤣

File: pageTests/admin/archive-a-ward-success.test.js (hunk 2 above)
Comment (jbevan4) [reply pageTests/admin/archive-a-ward-success.test.js:3]: I'm not sure, I didn't want to refactor all this in my tests. I'll bring the todo.

File: pageTests/admin/archive-a-ward-confirmation.test.js (hunk 1 above)
Comment (jbevan4) [reply pageTests/admin/archive-a-ward-confirmation.test.js:3]: See other resolved conversation.

**Pull request merged**